docs/sphinx/using/backends/hardware.rst

```diff
@@ -5,6 +5,129 @@ CUDA-Q supports submission to a set of hardware providers.
+
+For access to Infleqtion's neutral atom quantum computer, Sqale,
+`pre-registration <https://www.infleqtion.com/sqale-preregistration>`__ is now open.
+
+Setting Credentials
+`````````````````````````
+
+Programmers of CUDA-Q may access Infleqtion backends from either C++ or Python. Generate
+an API key from your `Superstaq account <https://superstaq.infleqtion.com/profile>`__ and export
+it as an environment variable:
+
+.. code:: bash
+
+  export SUPERSTAQ_API_KEY="superstaq_api_key"
+
+Submission from C++
+`````````````````````````
+
+To target quantum kernel code for execution on Infleqtion's backends,
+pass the flag ``--target infleqtion`` to the ``nvq++`` compiler.
+
+.. code:: bash
+
+    nvq++ --target infleqtion src.cpp
+
+This will take the API key and handle all authentication with, and submission to,
+the Infleqtion's QPU (or simulator). By default, quantum kernel code will be submitted to Infleqtion's Sqale
+simulator.
+
+To execute your kernels on a QPU, pass the ``--infleqtion-machine`` flag to the ``nvq++`` compiler
+to specify which machine to submit quantum kernels to:
+
+.. code:: bash
+
+    nvq++ --target infleqtion --infleqtion-machine cq_sqale_qpu src.cpp ...
+
+where ``cq_sqale_qpu`` is an example of a physical QPU.
+
+To run an ideal dry-run execution on the QPU, additionally pass the ``--infleqtion-method`` flag to the ``nvq++`` compiler:
+
+.. code:: bash
+
+    nvq++ --target infleqtion --infleqtion-machine cq_sqale_qpu --infleqtion-method dry-run src.cpp ...
+
+To noisily simulate the QPU instead, ``noise-sim`` to the ``--infleqtion-method`` flag like so:
+
+.. code:: bash
+
+    nvq++ --target infleqtion --infleqtion-machine cq_sqale_qpu --infleqtion-method noise-sim src.cpp ...
+
+Alternatively, to emulate the Infleqtion machine locally, without submitting through the cloud,
+you can also pass the ``--emulate`` flag to ``nvq++``. This will emit any target
+specific compiler diagnostics, before running a noise free emulation.
+
+.. code:: bash
+
+    nvq++ --emulate --target infleqtion src.cpp
+
+To see a complete example for using Infleqtion's backends, take a look at our :doc:`C++ examples <../examples/examples>`.
+
+Submission from Python
+`````````````````````````
+
+The target to which quantum kernels are submitted
+can be controlled with the ``cudaq::set_target()`` function.
+
+.. code:: python
+
+    cudaq.set_target('infleqtion')
+
+By default, quantum kernel code will be submitted to Infleqtion's Sqale
+simulator.
+
+To specify which Infleqtion QPU to use, set the :code:`machine` parameter.
+
+.. code:: python
+
+    cudaq.set_target("infleqtion", machine="cq_sqale_qpu")
+
+where ``cq_sqale_qpu`` is an example of a physical QPU.
+
+To run an ideal dry-run execution of the QPU, additionally set the ``method`` flag to ``"dry-run"``.
+
+.. code:: python
+
+    cudaq.set_target('infleqtion', machine="cq_sqale_qpu", method="dry-run")
```

Comment:
nit: Use double quotes throughout for consistency.
```cudaq.set_target("infleqtion", machine="cq_sqale_qpu", method="dry-run")```

Reply:
Thanks for the catch, fixed now